src/Tizen.Multimedia.MediaPlayer/Player/Player.Properties.cs

```diff
@@ -667,6 +676,57 @@ public float AudioPitch
             }
         }
 
+        /// <summary>
+        /// Gets or sets the default codec type of the audio decoder.
+        /// </summary>
+        /// <value>A <see cref="CodecType"/> that specifies the type.
+        /// The default codec type could be different depending on the device capability.</value>
+        /// <remarks>
+        /// <para>To set, the player must be in the <see cref="PlayerState.Idle"/> state.</para>
+        /// <para><see cref="CodecType.Sw"/> is supported basically.
+        /// If H/W audio codec type is not supported in some cases, S/W audio codec type could be used instead.</para>
```

Comment:
I think this statement is not necessary, if software codec is supported in all platform as described above statement.

Reply:
that sentence meant "even though an user set hw codec, it can be chagned to sw codec."

Reply:
You mean it can be changed automatically to software codec in case of hardware codec is not supported, right?
If so, when does 'CodecNotSupportedException` is thrown as described below?

Reply:
when the both hw codec and sw codec are not supported, it'll be thrown.

Reply:
software codec can be not supported, right?
If so, please fix the statement above.
`<see cref="CodecType.Sw"/> is supported basically.`

Reply:
Software codec can be also not supported, right?
If so, please fix the statement above.
<see cref="CodecType.Sw"/> is supported basically.